pl/mozorg/contribute/index.lang

```diff
@@ -242,156 +242,156 @@ Get Involved: Volunteer Opportunities at Mozilla
 
 
 ;To get started, tell us what interests you.
-To get started, tell us what interests you.
+Aby rozpocząć, powiedz nam, co cię interesuje.
 
 
 ;Choose one for now. You can always contribute to other areas later.
-Choose one for now. You can always contribute to other areas later.
+Wybierz jedno. Możesz pomóc w innych zakresach później.
 
 
 # This text is used for accessibility reasons, it's not visible
 ;Where would you like to get started?
-Where would you like to get started?
+Gdzie chciał(a)byś zacząć?
 
 
 ;Coding
-Coding
+Programowanie
 
 
 ;More about coding
-More about coding
+Więcej o programowaniu
 
 
 ;Contribute code to a whole range of Mozilla products, including Firefox and Firefox OS.
-Contribute code to a whole range of Mozilla products, including Firefox and Firefox OS.
+Twórz kod dla szerokiego zakresu produktów Mozilli, tj. Firefox i Firefox OS.
 
 
 ;Testing
-Testing
+Testowanie
 
 
 ;More about testing
-More about testing
+Więcej o testowaniu
 
 
 ;Help make the Mozilla products you use — and love — even better.
-Help make the Mozilla products you use — and love — even better.
+Pomóż nam czynić produkty Mozilli których używasz (i kochasz) jeszcze lepszymi.
```

Comment:
Zamiast „czynić”, można po prostu ulepszać.
Brakuje przecinka przed „których”.
„Love” w tym kontekście to bardziej „uwielbiać”.